using-kubecost/navigating-the-kubecost-ui/teams.md

```diff
@@ -23,79 +22,243 @@ persistentVolume:
 
-### Method 2: Disable SAML RBAC while leaving SAML enabled
+### Method 2: Enable SAML/OIDC without Teams, create teams, then enable Teams
 
-This method leaves additional authentication in place while SAML RBAC is disabled.
+This method leaves additional authentication in place while RBAC is disabled.
 
 ```yaml
-saml:
+oidc:
   enabled: true
   rbac:
     enabled: false
 ```
 
-Now, access the Teams page, and create your initial admin team. See the [Adding a team](teams.md#adding-a-team) section below for specific instructions for using the UI. Once your admin team has been created, reenable SAML RBAC authentication:
+Now, access the Teams page, and create your initial admin team. See the [Adding a team](teams.md#adding-a-team) section below for specific instructions for using the UI. Once your admin team has been created, reenable RBAC authentication:
 
 ```yaml
-saml:
+oidc:
   enabled: true
   rbac:
     enabled: true
 ```
 
 The members of your team are now able to add new members, or create new teams.
 
+### Method 3: Configure teams through Helm values
+
+{% hint style="warning" %}
+Configuring teams through Helm will disable configuration of teams through the UI.
+{% endhint %}
+
+
+Kubecost allows teams to be defined in Helm values obviating the need to create them in the UI. An example values snippet is shown below.
+
+```yaml
+ teams:
+   # teamsConfigMapName: teams-configmap-name
+   teamsConfig:
+     - id: ''
+       name: helm-team
+       roles:
+       - id: ''
+         name: helm-role
+         description: helm configured role
+         pages:
+           showOverview: true
+           showAllocation: true
+           showAsset: true
+           showCloudCost: true
+           showClusters: true
+           showExternalCosts: true
+           showNetwork: true
+           showCollections: true
+           showReports: true
+           showInsights: true
+           showActions: true
+           showAlerts: true
+           showBudgets: true
+           showAnomalies: true
+           showEfficiency: true
+           showSettings: true
+         permissions: admin
+         routes: []
+         allocationFilters:
+         - key: cluster
+           operator: ":"
+           value: cluster-one
+         assetFilters: []
+         cloudCostFilters: [] 
+       claims:
+         NameID: email@domain.com
+```
+
+Configure at least one team under `teamsConfig` with an associated role under the team's `roles`.
+
+Alternatively, a ConfigMap can be created manually, with `teamsConfigMapName` set to this ConfigMap's name.
+
+```yaml
+apiVersion: v1
+data:
+  rbac-teams-configs.json: '[json teams data]'
+kind: ConfigMap
+metadata:
+  name: <your-configmap-name>
+  namespace: <your-kc-namespace>
+
+Note that setting a manually-configured ConfigMap name will override `teamsConfig`.

```

Comment:
```suggestion
```
Note that setting a manually-configured ConfigMap name will override `teamsConfig`.
```